specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json

```diff
@@ -4983,6 +4983,16 @@
         "type": "object"
       }
     },
+    "FactoryPurviewConfiguration": {
```

Comment:
Please keep this property name the same as in Synapse as well: "PurviewConfiguration".

Reply:
Fixed